.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[Bug] ..."
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Create Custom Resource '....'
+3. Run command '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Desktop (please complete the following information):**
+ - Strimzi version: [e.g. master, 0.11.1]
+ - Installation method: [e.g. YAML files, Helm chart, OperatorHub.io]
+ - Kubernetes cluster: [e.g. Kubernetes 1.14, OpenShift 3.9]
+ - Infrastructure: [e.g. Amazon EKS, Minikube]
```

Comment:
Maybe we need OS version as well?

Reply:
OS version of what? The host OS should not matter. And the container OS should be always ours.